wwr/about.html

```diff
@@ -0,0 +1,90 @@
+    <link
+        href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+        rel="stylesheet">
+</head>
+
+<body>
+    <header>
+        <nav>
+            <img src="images/logo.jpg" alt="logo" width="150" height="150">
+            <ul>
+                <a href="index.html">Index</a>
+                <a href="about.html">About</a>
+                <a href="trips.html">Trips</a>
+                <a href="contact.html">Contact</a>
+
+            </ul>
+
+        </nav>
+    </header>
+    <main>
+        <div class="hero">
+            <img class="heroimg" src="images/heromelhor.jpg" alt="River">
+            <h1>WHITE WATER RAFTING</h1>
+        </div>
+        <article>
+            <img class="articleimg" src="images/hero.jpg" alt="random" width="150" height="150">
+            <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Quis accusantium reiciendis eum minima a
+                voluptates quo alias itaque cumque aliquid. Fugiat alias itaque dolorum nulla magnam. Facere
+                eligendi, quisquam, ipsum reprehenderit culpa modi sunt earum vitae natus quia at exercitationem non
+                provident, corrupti ducimus dolorum nostrum nisi laboriosam! Dolorum, tempora.</p>
+        </article>
+        <section>
+            <h2 class="titulo2">History</h2>
+            <div class="meiuca">
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Facere veniam corrupti officia quasi labore
+                    consequuntur nobis non explicabo adipisci rerum ex neque, voluptates animi laboriosam ab fugit
+                    assumenda omnis fuga velit ipsam dignissimos ratione architecto? Quod, provident? Consequatur sed
+                    officia, iusto quia impedit delectus quisquam ea reprehenderit placeat et dignissimos?</p>
+                <img class="middleimg" src="images/herocerto.jpg" alt="peoplewater">
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Magnam consectetur dicta illum dolorem, expedita
+                    odit, enim, in eveniet at maxime veritatis distinctio aut animi id nemo possimus ipsam. Eum nesciunt ab
+                    ipsa cum optio architecto odit quos. Beatae qui ea pariatur? Magnam molestias in quisquam cupiditate
+                    enim nulla, sequi quas!</p>
+            </div>
+        </section>
+
+
+        <section>
+            <h2 class="adventure">Adventure Awaits You!</h2>
+            <div class="pictures">
+                <img class="picture" src="images/hero.jpg" alt="boats">
+                <img class="picture" src="images/hero.jpg" alt="boats">
+                <img class="picture" src="images/hero.jpg" alt="boats">
+                <img class="picture" src="images/hero.jpg" alt="boats">
+                <img class="picture" src="images/hero.jpg" alt="boats">
```

Comment:
Each individual image should be nested inside a `figure` tag, along with a `figcaption` that describes the image. For example:

```html
<figure>
  <img src="images/wwr_6.jpg" alt="White water rafting 5">
  <figcaption>White water rafting 5</figcaption>
</figure>
```

Please never forget to read every instruction in the assignments. This is the 8th step of this activity's instructions.

[Activity Instructions](https://byui-cse.github.io/wdd130-ww-course/week04/prove-rafting-enhance.html#:~:text=Style%20the%20Adventures,Day%20Adventure%22%2C%20etc.)

![Image](https://github.com/user-attachments/assets/34c5ba95-361d-41fb-b887-d7296f35a576)